src/main/java/com/sda10/carrental/model/Revenue.java

```diff
@@ -0,0 +1,17 @@
+package com.sda10.carrental.model;
+
+import org.springframework.stereotype.Component;
+
+@Component
```

Comment:
Revenue nu este o componenta/un bean. Este un simplu POJO (PlainOldJavaObject) si locul lui este in pachetul .model